src/main/java/com/commercetools/sync/categories/CategorySync.java

```diff
@@ -395,35 +420,79 @@ private boolean isMissingCategory(@Nonnull final String categoryKey,
      *
      * @param createdCategories the set of created categories that needs to be processed.
      */
-    private void processCreatedCategories(@Nonnull final Set<Category> createdCategories) {
-        final int numberOfFailedCategories = newCategoryDrafts.size() - createdCategories.size();
+    private void processCreatedCategories(@Nonnull final Set<CategoryDraft> newCategories,
+                                          @Nonnull final Set<Category> createdCategories,
+                                          @Nonnull final Map<String, String> keyToIdCache) {
+        final int numberOfFailedCategories = newCategories.size() - createdCategories.size();
         statistics.incrementFailed(numberOfFailedCategories);
```

Comment:
i recommend to update statistics in `createCategories` method - it seems to me more the right place to do that. wdyt?

Reply:
1f8be26f6

Reply:
Thanks 👍 
You can also move `statistics.incrementCreated` to the `createCategories`. Sorry i just saw it.

Reply:
87481216e